core/src/test/php/net/xp_framework/unittest/core/NamespaceAliasTest.class.php

```diff
@@ -44,19 +45,32 @@ public function defined_interface_exists() {
 
   #[@test]
   public function autoloaded_class_exists() {
-    new \net\xp_framework\unittest\core\NamespaceAliasAutoloadedFixture();    // Triggers autoloader
+    new NamespaceAliasAutoloadedFixture();              // Triggers autoloader
     $this->assertTrue(class_exists('net\xp_framework\unittest\core\NamespaceAliasAutoloadedFixture', false));
   }
 
   #[@test]
   public function autoloaded_namespaced_class_exists() {
-    new \net\xp_framework\unittest\core\NamespaceAliasAutoloadedNamespacedFixture();    // Triggers autoloader
+    new NamespaceAliasAutoloadedNamespacedFixture();    // Triggers autoloader
     $this->assertTrue(class_exists('net\xp_framework\unittest\core\NamespaceAliasAutoloadedNamespacedFixture', false));
   }
 
   #[@test]
   public function autoloaded_fq_class_exists() {
-    new \net\xp_framework\unittest\core\NamespaceAliasAutoloadedFQFixture();    // Triggers autoloader
+    new NamespaceAliasAutoloadedFQFixture();            // Triggers autoloader
     $this->assertTrue(class_exists('net\xp_framework\unittest\core\NamespaceAliasAutoloadedFQFixture', false));
   }
+
+  #[@test]
+  public function namespaced_classes_aliased_to_short_name_when_loaded_via_uses() {
+    uses('net.xp_framework.unittest.core.NamespaceAliasAutoloadedNamespacedFixture');
+    $this->assertTrue(class_exists('NamespaceAliasAutoloadedNamespacedFixture', false));
+  }
+
+  #[@test]
+  public function namespaced_classes_not_aliased_twice_when_loaded_via_uses() {
```

Comment:
Hrm, yes, and no :grinning: 

* Yes, because you'd also want to test that `uses("my.namespaced.Date");` doesn't overwrite util.Date
* No, because it is called "not aliased twice", meaning two calls to uses (e.g. from different files) don't make `class_alias()` run twice (and throw a warning when doing so).

It might be best to add a separate test case, although it boils down to the same code in the end.